boots/fields/widgets.py

```diff
@@ -181,3 +179,20 @@ def format_output(self, rendered_widgets):
             os.path.realpath(__file__)))
         data = {'min': rendered_widgets[0], 'max': rendered_widgets[1]}
         return render_to_string('date_range.html', data, dirs=[template_path])
+
+
+class RadioWidget(forms.RadioSelect):
+    template = "radio_input.html"
+
+    def __init__(self, radio, *args, **kwargs):
+        self.radio = radio
+        super(RadioWidget, self).__init__(**kwargs)
+
+    def render(self, name, value, attrs=None, choices=()):
```

Comment:
It seems choices gets passed along. That would seem to be the more django way of doing it. Is there a way we can use the standard choice tuple of tuples?